product/roundhouse.console/roundhouse.console.csproj

```diff
@@ -68,115 +47,93 @@
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Windows.Installer.3.1">
-      <Visible>False</Visible>
-      <ProductName>Windows Installer 3.1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
+    <ProjectReference Include="..\roundhouse.databases.access\roundhouse.databases.access.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse.databases.mysql\roundhouse.databases.mysql.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse.databases.oracle\roundhouse.databases.oracle.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse.databases.postgresql\roundhouse.databases.postgresql.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse.databases.sqlite\roundhouse.databases.sqlite.csproj" ExcludeAssets="All"  />
+    <ProjectReference Include="..\roundhouse.databases.sqlserver2000\roundhouse.databases.sqlserver2000.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse.databases.sqlserverce\roundhouse.databases.sqlserverce.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse.databases.sqlserver\roundhouse.databases.sqlserver.csproj" ExcludeAssets="All" />
+    <ProjectReference Include="..\roundhouse\roundhouse.csproj" ExcludeAssets="All" />
   </ItemGroup>
   <ItemGroup>
     <None Include="..\..\docs\logo\roundhouse.ico">
       <Link>roundhouse.ico</Link>
     </None>
-    <None Include="app.config" />
-    <None Include="Statements.txt" />
-    <None Include="targets\Microsoft.Application.targets" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project=".\targets\Microsoft.Application.targets" />
-  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
+
+  <!-- Merge properties and target -->
+  <PropertyGroup>
+     <ILMergeDir>..\..\code_drop\merge\</ILMergeDir>
+     <ILMergeTarget>$(ILMergeDir)$(AssemblyName).exe</ILMergeTarget>
+  </PropertyGroup>
+
+  <Target Name="ILMerge" BeforeTargets="Pack">
+    <ItemGroup>
+      <ILMergeAssemblies Include="$(OutputPath)*.dll" />
+    </ItemGroup>
+    <PropertyGroup>
+      <IgnoreFile>..\..\build.custom\ilmerge.internalize.ignore.txt</IgnoreFile>
+      <ILMerge>..\..\lib\ILMerge\ILMerge.exe</ILMerge>

```

Comment:
Where does this resource come from?

Reply:
Which resource? ILMerge.exe?

Reply:
`<ILMerge>` - typically if you use things from MSBuild that are not included, it can cause failures. It's good to know if this is build into MsBuild or requires a community pack or something.

Reply:
No, no, <ILMerge> is not a task. It is just a property. Maybe I should have called it <ILMergeExe> or something.

Reply:
I see it now... property group. My bad...